homeassistant/components/stream/__init__.py

```diff
@@ -219,6 +141,42 @@ def redact_credentials(url: str) -> str:
     return str(yurl.update_query(redacted_query_params))
 
 
+def _convert_stream_options(
+    hass: HomeAssistant,
+    stream_source: str,
+    stream_options: Mapping[str, str | bool | float],
+) -> tuple[dict[str, str], StreamSettings]:
+    """Convert options from stream options into PyAV options and stream settings."""
+    if DOMAIN not in hass.data:
+        raise HomeAssistantError("Stream integration is not set up.")
```

Comment:
Would be nice to add coverage for this line

Reply:
I went ahead and added some tests for this now.